test/e2e/util/util.go

```diff
@@ -1461,10 +1462,22 @@ func EnsurePodsWithEmptyDirPVsHaveSafeToEvictAnnotations(t *testing.T, ctx conte
 
 			if labelKey == "" || labelValue == "" {
 				// if the Key/Value are empty we assume that the pod is not in the auditedList,
-				// if that's the case the annotation should not exists in that pod.
+				// if that's the case the annotation should not exists in that pod (except for tmp-dir which is in every pod by default).
 				// Then continue to the next pod
-				g.Expect(pod.Annotations[suppconfig.PodSafeToEvictLocalVolumesKey]).To(BeEmpty(), "the pod  %s is not in the audited list for safe-eviction and should not contain the safe-to-evict-local-volume annotation", pod.Name)
-				continue
+				hasTmpDir := false
+				safe2EvictVolumes := strings.Split(pod.Annotations[suppconfig.PodSafeToEvictLocalVolumesKey], ",")
+				safe2EvictVolumes = slices.DeleteFunc(safe2EvictVolumes, func(s string) bool {
+					if s == "" {
```

Comment:
nit:
```
return s == "" || s == CONSTANT_FOR_tmp_dir
```

Reply:
So I intentionally separated out this condition since I need to distinguish later if the pod has tmpdir in the annotation, but we need to remove both the tmpdir annotation and any empty value from the safe2EvictVolumes slice for the check below to function properly